admin-dev/themes/new-theme/js/pages/feature-flag/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+import ConfirmModal from '@components/modal';
+
+const {$} = window;
+
+$(() => {
+  const $submitButton = $('#submit-btn-feature-flag');
+  const $form = $('#feature-flag-form');
+
+  $submitButton.on('click', (event) => {
+    event.preventDefault();
+
+    const modal = new ConfirmModal(
+      {
+        id: 'modal-confirm-submit-feature-flag',
+        confirmTitle: $submitButton.data('modal-title'),
+        confirmMessage: $submitButton.data('modal-message'),
```

Comment:
If you wan an example of plural wording we used some here:
https://github.com/PrestaShop/PrestaShop/blob/de8a73cb0bc8484e426771b938a11f58628ebfda/src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php#L81

It meets your use case because the pluralization is not handled by Twig (or Symfony) here, because the count is dynamic and needs to be handled by the JS, so it's a bit more complicated.

That's why we don't use `transChoice` in our FormType but only `trans` this way the wording is translated but it keeps it plural syntax (separated by a `|`). Then in our case we use a Vue component with VueI18n so we have a `tc` function (equivalent to `transChoice`)

However in your case you use jQuery or native js so I guess we'll need to:
- implement our own javascript `transChoice` function
- use a library which already does it better than us ^^

Reply:
I can't find a vanilla lib using delimiter like the pipe one, maybe it's easier to create a helper function, exploding the text and conditioning with the number?

Reply:
Transchoice is more complex than that because in some languages there are different plural forms depending on the number of items. See https://unicode-org.github.io/icu/userguide/format_parse/messages/#complex-argument-types

Reply:
As I think handling the plural is likely to increase the quantity of JS for quite an amount, I mentioned in PR body it would be done in another iteration. A dedicated PR will be done for it, so we can look for the best implementation while not blocking this PR 😊